### PR TITLE
add post sonar scans

### PR DESCRIPTION
**Pull request by chrisahl**
Signed-off-by: Chris Ahl <cahl@redhat.com>

For some items in issue https://github.com/open-cluster-management/backlog/issues/13385

### CODE REVIEW

File: ci-operator/jobs/open-cluster-management/clusterlifecycle-state-metrics/open-cluster-management-clusterlifecycle-state-metrics-main-postsubmits.yaml [hunk 1]
```diff
@@ -135,3 +135,42 @@ postsubmits:
...
+        command:
+        - make
+        env:
+        - name: ARTIFACT_DIR
+          value: $(ARTIFACTS)
+        - name: GOPROXY
+          value: proxy.golang.org,direct
+        - name: SELF
+          value: make -f /opt/build-harness/Makefile.prow
+        image: registry.ci.openshift.org/open-cluster-management/builder:go1.15-linux
```
Comment (itdove) [inline ci-operator/jobs/open-cluster-management/clusterlifecycle-state-metrics/open-cluster-management-clusterlifecycle-state-metrics-main-postsubmits.yaml:161]: I think it should be 1.16 as the dockerfile.prow is using go1.16

File: ci-operator/jobs/open-cluster-management/clusterlifecycle-state-metrics/open-cluster-management-clusterlifecycle-state-metrics-release-2.3-postsubmits.yaml [hunk 2]
```diff
@@ -140,3 +140,42 @@ postsubmits:
...
+        command:
+        - make
+        env:
+        - name: ARTIFACT_DIR
+          value: $(ARTIFACTS)
+        - name: GOPROXY
+          value: proxy.golang.org,direct
+        - name: SELF
+          value: make -f /opt/build-harness/Makefile.prow
+        image: registry.ci.openshift.org/open-cluster-management/builder:go1.15-linux
```
Comment (itdove) [inline ci-operator/jobs/open-cluster-management/clusterlifecycle-state-metrics/open-cluster-management-clusterlifecycle-state-metrics-release-2.3-postsubmits.yaml:166]: Same 1.16

**Review by itdove — COMMENTED** [review]
go 1.16 comments, Thx

File: ci-operator/jobs/open-cluster-management/clusterlifecycle-state-metrics/open-cluster-management-clusterlifecycle-state-metrics-main-postsubmits.yaml (hunk 1 above)
Comment (chrisahl) [reply ci-operator/jobs/open-cluster-management/clusterlifecycle-state-metrics/open-cluster-management-clusterlifecycle-state-metrics-main-postsubmits.yaml:161]: updated

File: ci-operator/jobs/open-cluster-management/clusterlifecycle-state-metrics/open-cluster-management-clusterlifecycle-state-metrics-release-2.3-postsubmits.yaml (hunk 2 above)
Comment (chrisahl) [reply ci-operator/jobs/open-cluster-management/clusterlifecycle-state-metrics/open-cluster-management-clusterlifecycle-state-metrics-release-2.3-postsubmits.yaml]: updated

**Pull request merged**